templates/kafka/0/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+zk:
```

Comment:
Sorry, I just read the image name and didn't go digging into it because I've had this discussion with apparently someone else before.  Still, those all seem like good things to put in a PR to the rancher-catalog zookeeper; we take PRs there too.  Or a rawminds-zookeper-thats-awesomer-than-ranchers template in here.

I  just don't like the idea of having a separate re-implementation of the same service in every template that happens to need common basic services like this.  If you make a, say, Solr template next is it going to have another copy of exactly the same zookeeper service in it?